### PR TITLE
Fixed W-12496812 p1 bug & added empty screen UI

### PR DESCRIPTION
**Pull request by ibenitamar**

### CODE REVIEW

**Review by maliroteh-sf — CHANGES_REQUESTED** [review]
@ibenitamar This is an open-source repo and as such we should not be referencing any internal work item numbers in PR title or commit messages. Please close this PR and open a new one with appropriate commit message & PR title.

File: RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingResourceAssignment/mobileAppointmentBookingResourceAssignment.html [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+                        <lightning-radio-group name="resourceAssignment"
+                                aria-label={resourceAssignmentTitle}
+                                options={options}
+                                value={value}
+                                type="radio"
+                                onchange={handleAssignmentMethodChange}
+                                >
+                        </lightning-radio-group>
+                    </div>
+                </template>
```
Comment (maliroteh-sf) [inline RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingResourceAssignment/mobileAppointmentBookingResourceAssignment.html:20]: fix indentation/formating

File: RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingResourceAssignment/mobileAppointmentBookingResourceAssignment.html [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+                <template lwc:else>
+                    <div>
+                        <lightning-radio-group name="resourceAssignment"
+                                aria-label={resourceAssignmentTitle}
+                                options={options}
+                                value={value}
+                                type="radio"
+                                onchange={handleAssignmentMethodChange}
+                                >
```
Comment (maliroteh-sf) [inline RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingResourceAssignment/mobileAppointmentBookingResourceAssignment.html:17]: `>` should be at the end of the previous line: `onchange={handleAssignmentMethodChange}>`

**Comment by ibenitamar** [comment]
@maliroteh-sf done thanks.

**Pull request closed without merging**